packages/jellyfish-json/src/remap.ts

```diff
@@ -80,49 +123,46 @@ function remapLosslessObj (losslessObj: any, precision: PrecisionMapping): any {
 }
 
 function getAction (value: any, type: Precision | PrecisionMapping): MappingAction {
+  const mappingAction = getPrecisionMappingAction(value, type)
+
+  if (mappingAction === MappingAction.NONE) {
+    return getDeeplyMappingAction(value, type)
+  }
+
+  return mappingAction
+}
+
+function getPrecisionMappingAction (value: any, type: Precision | PrecisionMapping): MappingAction {
```

Comment:
it's okay to keep the if condition long, don't care the codeclimate warning on this.